index.md

```diff
@@ -230,6 +230,75 @@ The example below applies a `margin-right` style to the first input.
   <<< .vitepress/docs/components/HorizontalForm.vue
 </SplitTab>
 
+### Prop: schemaRowClasses
+
+The philosophy is `Bring Your Own Component` and there will be moments when you already have your own css or a Framework UI you are using dictates that you need to use specific classes.
+
+For those opinated situations `schemaRowClasses` allows you to complement the `schema-row` class that Formvuelate provides.
+
+The prop accepts `[String, Object, Array]`, since any of those are valid ways to set CSS in Vue.
+
+```html
+<SchemaForm
+  :schemaRowClasses="my-custom-class-a my-custom-class-b"
+  :schema="mySchema"
+/>
+```
+When the `Form` html element is rendered, there will be a `div` element with those classes:
+
+```html
+<form>
+  <div class="schema-row custom-class-a custom-class-b">
+    Fields
+  </div>
+</form>
+```
+:::tip
+You might find scenarios where the Field must have a class that depends on the class applied to `schemaRowClasses`. Fortunately, Formvuelate has you covered.
+
+For instance if you need this:
+```html
```

Comment:
This whole bit with the tip until the end is nice, but I feel it overexplains component composition which is not directly related to what FVL does. Let's remove this bit and keep it short and clean